src/neptune/metadata_containers/metadata_container.py

```diff
@@ -143,6 +148,37 @@ def __init__(
 
         self._startup(debug_mode=mode == Mode.DEBUG)
 
+        os.register_at_fork(after_in_child=self._handle_fork_in_child, after_in_parent=self._handle_fork_in_parent)
+
+    """
```

Comment:
Can we move this comment to `reset_internal_ssl_state` as well?

Reply:
Done.